src/pages/[chapterId]/[verseId]/index.tsx

```diff
@@ -88,7 +88,7 @@ export const getStaticProps: GetStaticProps = async ({ params, locale }) => {
   if (isVerse) {
     apiParams = { ...apiParams, ...{ page: verseIdOrRange, perPage: 1 } };
   } else {
-    [from, to] = getToAndFromFromRange(verseIdOrRange);
+    [from, to] = getToAndFromFromRange(verseIdOrRange).map((ayah) => `${chapterIdOrSlug}:${ayah}`);
```

Comment:
This won't work when there is a `slug` also `getToAndFromFromRange` is used in other places not only here.

Reply:
Also `from` and `to` are being sent to BE from other places too e.g. `getAdvancedCopyRawResult`